app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Listing Users</h1>
+<% @users.each do |user| %>
+  <div class="user">
+    <div class="user-photo">
+      <img src="<%= user.photo %>" alt="User Photo">
```

Comment:
![image](https://github.com/torobucci/blog_app/assets/93039359/6b19af3c-62e3-4518-90e4-ae32020305c1)

- [ ] It would be nice if you fix the image on all the pages because as you can see it is overlapping with the other contents :fearful: KIndly make it follow the wireframe to make your design look professional :+1: